apps/blur/test.cpp

```diff
@@ -14,6 +14,8 @@
 
 double t;
 
+#pragma GCC diagnostic ignored "-Wcast-qual"
```

Comment:
I think I originally did this because I was experimenting with using ``#pragma GCC diagnostic push`` and  ````#pragma GCC diagnostic pop``.

I'll rewrite setting this flag in the ``CMakeLists.txt`` file